### PR TITLE
Adds Hotel Station

### PR DESCRIPTION
**Pull request by PubliclyExecutedPig**
## Short description

Adds Hotel Station, a midpop station made for starlight. wip rn
## Why we need to add this
Adds variety to the map pool and gives starlight some more exclusive stuff 

## Media (Video/Screenshots)

Hotel-0

## Checks

- [X] I do not require assistance to complete the PR.
- [X] Before posting/requesting review of a PR, I have verified that the changes work.
- [X] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [X] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**

 🆑 PubliclyExecutedPig
- add: Added Hotel Station

### CODE REVIEW

**Comment by PubliclyExecutedPig** [comment]
still wip, will post updates here

**Comment by PubliclyExecutedPig** [comment]
map renderer isnt working :blunt: 
image

**Comment by Rinary1** [comment]
@ss14-Starlight

**Comment by PubliclyExecutedPig** [comment]
i forgot to add signs but now im done, except for fixing this error lol

**Comment by PubliclyExecutedPig** [comment]
drafting for a sec bc i wanna add decals and stuff

**Comment by JustAdler** [comment]
> drafting for a sec bc i wanna add decals and stuff

maybe also decal arrows for hop line? same from cargo

**Comment by PubliclyExecutedPig** [comment]
> > drafting for a sec bc i wanna add decals and stuff
> 
> maybe also decal arrows for hop line? same from cargo

done, just the map thing isnt working bc of some bug to do with missing red bruise pack textures or something

**Comment by Rinary1** [comment]
merge master(Starlight)

**Comment by PubliclyExecutedPig** [comment]
> merge master(Starlight)

done

File: Resources/Prototypes/_StarLight/Maps/hotel.yml
Comment (Rinary1) [inline Resources/Prototypes/_StarLight/Maps/hotel.yml:1]: Law department? Or it didn't mapped?
+ Add it to map pool(/Pools/Default.yml)

**Review by Rinary1 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_StarLight/Maps/hotel.yml
Comment (PubliclyExecutedPig) [reply Resources/Prototypes/_StarLight/Maps/hotel.yml:1]: There's a magistrate office and a courtroom, also a small Internal affairs agent office

Comment (PubliclyExecutedPig) [reply Resources/Prototypes/_StarLight/Maps/hotel.yml:1]: I'll add it when I'm home

File: Resources/Prototypes/_StarLight/Maps/hotel.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 - type: gameMap
   id: Hotel
-  mapName: 'Hotel'
+  mapName: 'StarlightHotel'
```
Comment (Rinary1) [inline Resources/Prototypes/_StarLight/Maps/hotel.yml]: ????

**Review by Rinary1 — CHANGES_REQUESTED** [review]

**Pull request merged**